package/libs/libusb/Makefile

```diff
@@ -18,7 +18,6 @@ PKG_SOURCE_URL:=\
 PKG_HASH:=75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157
 
 PKG_INSTALL:=1
-PKG_BUILD_PARALLEL:=0
```

Comment:
and this one. I think all of these should be a follow up PR later

Reply:
Can't reproduce any build issues